src/com/projectkorra/projectkorra/earthbending/EarthGrab.java

```diff
@@ -203,20 +203,20 @@ public void trap() {
 			this.mHandler = new MovementHandler(this.target, this);
 			this.mHandler.stop(Element.EARTH.getColor() + "* Trapped *");
 
-			if (this.target instanceof Player || this.target instanceof Zombie || this.target instanceof Skeleton) {
-				final ItemStack legs = new ItemStack(Material.LEATHER_LEGGINGS);
-				final LeatherArmorMeta legmeta = (LeatherArmorMeta) legs.getItemMeta();
-				legmeta.setColor(Color.fromRGB(EarthArmor.getColor(m)));
-				legs.setItemMeta(legmeta);
-
-				final ItemStack feet = new ItemStack(Material.LEATHER_BOOTS);
-				final LeatherArmorMeta footmeta = (LeatherArmorMeta) feet.getItemMeta();
-				footmeta.setColor(Color.fromRGB(EarthArmor.getColor(m)));
-				feet.setItemMeta(footmeta);
-
-				final ItemStack[] pieces = { (this.target.getEquipment().getArmorContents()[0] == null || this.target.getEquipment().getArmorContents()[0].getType() == Material.AIR) ? feet : null, (this.target.getEquipment().getArmorContents()[1] == null || this.target.getEquipment().getArmorContents()[1].getType() == Material.AIR) ? legs : null, null, null };
-				this.armor = new TempArmor(this.target, 36000000L, this, pieces);
-			}
+//			if (this.target instanceof Player || this.target instanceof Zombie || this.target instanceof Skeleton) {
+//				final ItemStack legs = new ItemStack(Material.LEATHER_LEGGINGS);
+//				final LeatherArmorMeta legmeta = (LeatherArmorMeta) legs.getItemMeta();
+//				legmeta.setColor(Color.fromRGB(EarthArmor.getColor(m)));
+//				legs.setItemMeta(legmeta);
+//
+//				final ItemStack feet = new ItemStack(Material.LEATHER_BOOTS);
+//				final LeatherArmorMeta footmeta = (LeatherArmorMeta) feet.getItemMeta();
+//				footmeta.setColor(Color.fromRGB(EarthArmor.getColor(m)));
+//				feet.setItemMeta(footmeta);
+//
+//				final ItemStack[] pieces = { (this.target.getEquipment().getArmorContents()[0] == null || this.target.getEquipment().getArmorContents()[0].getType() == Material.AIR) ? feet : null, (this.target.getEquipment().getArmorContents()[1] == null || this.target.getEquipment().getArmorContents()[1].getType() == Material.AIR) ? legs : null, null, null };
+//				this.armor = new TempArmor(this.target, 36000000L, this, pieces);
+//			}

```

Comment:
What was the dupe glitch that this caused?

Reply:
When a mob was EarthGrabbed, if it picked up armor it would replace the armor created by the ability, then on death would drop two copies if they died while in EarthGrab